api/Slotable.json

```diff
@@ -0,0 +1,107 @@
+{
+  "api": {
+    "Slotable": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Slotable",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "assignedSlot": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Slotable/assignedSlot",
+          "support": {
+            "webview_android": {
+              "version_added": "53"
+            },
+            "chrome": {
+              "version_added": "53"
+            },
+            "chrome_android": {
+              "version_added": "53"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "58",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.webcomponents.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": {
+              "version_added": true
```

Comment:
You missed the information for Firefox for Android, this should be the same as in Firefox for Desktop.

Reply:
Right, my bad